src/drafts/MarkdownEditor/MarkdownEditor.features.stories.tsx

```diff
@@ -0,0 +1,437 @@
+import {DiffIcon, PlusIcon} from '@primer/octicons-react'
+import React, {Meta} from '@storybook/react'
+import {useRef, useState} from 'react'
+import BaseStyles from '../../BaseStyles'
+import Box from '../../Box'
+import MarkdownEditor, {Emoji, Mentionable, Reference, SavedReply} from '.'
+import ThemeProvider from '../../ThemeProvider'
+
+const meta: Meta = {
+  title: 'Drafts/Components/MarkdownEditor/Features',
+  decorators: [
+    Story => {
+      return (
+        <ThemeProvider>
+          <BaseStyles>
+            <Box sx={{maxWidth: 800}}>{Story()}</Box>
+          </BaseStyles>
+        </ThemeProvider>
+      )
+    },
+  ],
```

Comment:
```suggestion
```

You probably won't need these as we set them in the story config ([reference](https://github.com/primer/react/blob/main/src/utils/story-helpers.tsx))

To set a max-width to the box, I think we can do it in the story itself. I'll leave a comment there too. Let me know if you have any concern or any more clarifications you need 😊